code/system/application/knowledge/include/linguistics/v2vcam/v2vcam.hpp

```diff
@@ -82,9 +82,14 @@ class V2vCam
   int32_t GetYawRateConfidence() const;
   int32_t GetVehicleRole() const;
 
+  bool m_initialised;
   std::ofstream m_sendLog;
   std::ofstream m_receiveLog;
```

Comment:
Perhaps, since we don't have specialized messages for the V2V messages. We should perhaps have this...